### PR TITLE
Fix compile errors in nbla_cli build

### PR DESCRIPTION
**Pull request by hakuturu583**
I solved compile errors described in the issue #146 .

### CODE REVIEW

**Comment by hakuturu583** [comment]
I build nnabla and ext-cuda like this.
(https://gist.github.com/hakuturu583/b44d1c7861ae3a450fe9965d055925ba)

**Comment by TE-TakuyaNarihira** [comment]
Thanks for the PR! @TE-NaokiIde could you review this?

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 /build_*/
 /build-multi-gpu_*/
+build
```
[the remark this answers is not in the document]
Comment (TE-TakuyaNarihira) [reply .gitignore]: Also, please squash the change into 28bcad25c7e51c16909ac5ee9a25548a62bfadab.

[the remark this answers is not in the document]
Comment (hakuturu583) [reply .gitignore]: I fixed!! Thanks!

[the remark this answers is not in the document]
Comment (TE-NaokiIde) [reply .gitignore]: Thanks!

**Pull request merged**